packages/expo/hooks/lib/report-build.js

```diff
@@ -0,0 +1,10 @@
+const reportBuild = require('bugsnag-build-reporter')
+
+module.exports = async (apiKey, manifest, projectRoot) => {
+  const { revisionId, version } = manifest
+  await reportBuild({
+    apiKey,
+    appVersion: version,
+    metadata: { [`bundle@${new Date().toISOString()}`]: revisionId }
```

Comment:
This means you get a list of `revisionId`s that were shipped under this app version on the release page:

![image](https://user-images.githubusercontent.com/609579/54915581-6929ad00-4eef-11e9-9af8-248e6e58a7a6.png)